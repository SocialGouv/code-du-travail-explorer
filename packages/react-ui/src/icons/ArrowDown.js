import React from "react";

export const ArrowDown = () => (
  <svg viewBox="0 0 18 18" fill="none">
    <title>arrow down</title>
    <path
      d="M1 6L9 12L17 6"
      stroke="currentColor"
      strokeWidth="1.25"
      strokeLinecap="round"
      strokeLinejoin="round"
    />
  </svg>
);
