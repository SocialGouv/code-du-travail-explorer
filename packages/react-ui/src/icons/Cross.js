import React from "react";

export const Cross = () => (
  <svg viewBox="0 0 8 8" fill="none">
    <title>cross</title>
    <path
      d="M0.666748 7.33342L4.00008 4.00008M7.33342 0.666748L4.00008 4.00008M4.00008 4.00008L0.666748 0.666748M4.00008 4.00008L7.33342 7.33342"
      stroke="currentColor"
      strokeLinecap="round"
    />
  </svg>
);
