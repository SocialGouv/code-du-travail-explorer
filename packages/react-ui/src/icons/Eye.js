import React from "react";

export const Eye = () => (
  <svg viewBox="0 0 36 30" fill="none">
    <title>Eye</title>
    <path
      d="M18.6416 4.83333V7.43333C18.6416 7.80335 18.3533 8.09167 17.9832 8.09167C17.6132 8.09167 17.3249 7.80335 17.3249 7.43333V4.83333C17.3249 4.46332 17.6132 4.175 17.9832 4.175C18.3533 4.175 18.6416 4.46332 18.6416 4.83333Z"
      fill="currentColor"
    />
    <path
      d="M13.6319 8.66615L13.6181 8.67041L13.6051 8.67689C13.5805 8.68919 13.5545 8.69174 13.4834 8.69174C13.2052 8.69174 12.9316 8.48705 12.8172 8.18078L12.1537 5.65965C12.0757 5.28676 12.2855 4.94542 12.6276 4.83629C13.0005 4.76016 13.3411 4.97162 13.4485 5.31495L14.1131 7.84051C14.1916 8.21603 13.9783 8.55957 13.6319 8.66615Z"
      fill="currentColor"
    />
    <path
      d="M9.59523 10.3487L9.59507 10.3484L9.58633 10.3543C9.5265 10.3941 9.39911 10.4249 9.2834 10.4249C9.05275 10.4249 8.83767 10.2974 8.69734 10.1049L7.40187 7.84614L7.40188 7.84614L7.40123 7.84503C7.21693 7.52909 7.32116 7.136 7.63825 6.95103C7.95402 6.76683 8.34684 6.87085 8.53194 7.18753L9.8316 9.4536L9.83159 9.45361L9.83224 9.45472C10.0165 9.77066 9.91231 10.1637 9.59523 10.3487Z"
      fill="currentColor"
    />
    <path
      d="M6.12626 12.0904L6.12608 12.0905L6.13293 12.0967C6.3852 12.3238 6.39507 12.7407 6.12626 13.0096C6.00035 13.1355 5.81617 13.1916 5.65 13.1916C5.50818 13.1916 5.34903 13.1199 5.20001 13.0025L3.37374 11.1762L3.37391 11.1761L3.36707 11.1699C3.1148 10.9429 3.10493 10.5259 3.37374 10.257L3.37391 10.2572L3.38008 10.2504C3.60712 9.9981 4.02411 9.98823 4.29292 10.257L6.12626 12.0904Z"
      fill="currentColor"
    />
    <path
      d="M23.1141 8.18878L23.1141 8.18877L23.1136 8.19087C23.0338 8.51009 22.7727 8.69165 22.4834 8.69165C22.474 8.69165 22.4635 8.68945 22.4172 8.67915C22.3823 8.67139 22.335 8.66148 22.2778 8.65893C21.9098 8.54276 21.7172 8.17668 21.8173 7.85144L21.8174 7.85149L21.8193 7.84452L22.4855 5.31305C22.595 4.93299 22.9679 4.73216 23.2986 4.83391L23.2986 4.83394L23.302 4.83491C23.6831 4.94381 23.8847 5.31734 23.7828 5.64851L23.7826 5.64847L23.7808 5.65544L23.1141 8.18878Z"
      fill="currentColor"
    />
    <path
      d="M26.1344 9.42151L26.135 9.4204L27.4347 7.15433C27.6198 6.83764 28.0126 6.73363 28.3284 6.91783C28.6531 7.10726 28.7394 7.52881 28.5732 7.76631L28.5687 7.77277L28.5648 7.77961L27.2648 10.0463L27.2647 10.0463L27.2636 10.0483C27.1299 10.289 26.9241 10.3917 26.6832 10.3917C26.5817 10.3917 26.4767 10.3669 26.3666 10.3127C26.0533 10.1266 25.951 9.73586 26.1344 9.42151Z"
      fill="currentColor"
    />
    <path
      d="M29.8405 12.9764L29.8407 12.9762L29.8339 12.97C29.5816 12.743 29.5717 12.326 29.8405 12.0572L31.6739 10.2238C31.9389 9.95885 32.3614 9.95885 32.6264 10.2238L32.6262 10.224L32.6331 10.2302C32.8853 10.4572 32.8952 10.8742 32.6264 11.143L30.7931 12.9764C30.665 13.1044 30.5126 13.1584 30.3168 13.1584C30.121 13.1584 29.9686 13.1044 29.8405 12.9764Z"
      fill="currentColor"
    />
    <path
      d="M31.3685 17.774L31.368 17.7748C28.6012 22.2544 23.4921 25.0584 17.9835 25.0584C12.4751 25.0584 7.36623 22.2545 4.59939 17.742L4.59859 17.7407C4.4674 17.5308 4.4674 17.2694 4.5986 17.0595L4.59908 17.0587C7.36589 12.5791 12.475 9.7751 17.9835 9.7751C23.4919 9.7751 28.6008 12.579 31.3677 17.0916L31.3685 17.0928C31.4997 17.3028 31.4997 17.5641 31.3685 17.774ZM6.00518 17.3355L5.93902 17.4334L6.00518 17.5314C8.60575 21.3816 13.1279 23.7418 17.9835 23.7418C22.84 23.7418 27.3618 21.3476 29.9619 17.498L30.028 17.4001L29.9619 17.3021C27.3613 13.452 22.8392 11.0918 17.9835 11.0918C13.127 11.0918 8.60528 13.486 6.00518 17.3355Z"
      fill="currentColor"
    />
    <path
      d="M13.9917 17.4001C13.9917 15.1967 15.78 13.4084 17.9833 13.4084C20.1867 13.4084 21.975 15.1967 21.975 17.4001C21.975 19.6034 20.1867 21.3917 17.9833 21.3917C15.78 21.3917 13.9917 19.6034 13.9917 17.4001ZM15.3083 17.4001C15.3083 18.8634 16.52 20.0751 17.9833 20.0751C19.4466 20.0751 20.6583 18.8634 20.6583 17.4001C20.6583 15.9367 19.4466 14.7251 17.9833 14.7251C16.52 14.7251 15.3083 15.9367 15.3083 17.4001Z"
      fill="currentColor"
    />
  </svg>
);
