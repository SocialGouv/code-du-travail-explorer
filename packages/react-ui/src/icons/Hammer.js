import React from "react";

export const Hammer = () => (
  <svg viewBox="0 0 36 30" fill="none">
    <path
      d="M15.5036 0.00132414C15.121 0.0177034 14.7425 0.189019 14.4639 0.502219L13.4542 1.66054C12.9392 2.23947 12.9294 3.10958 13.409 3.69542L9.05406 8.61045C8.45009 8.15638 7.59759 8.23216 7.07998 8.81394L6.05528 9.97226C5.49799 10.5987 5.54071 11.5713 6.1457 12.148L11.7365 17.4856C12.3414 18.0623 13.2888 18.0181 13.8461 17.3916L14.8708 16.249C15.3909 15.6646 15.3796 14.7685 14.8859 14.1828L16.1065 12.8054C18.4962 15.6425 22.0049 19.4512 25.0727 22.7136C26.6702 24.4123 28.1454 25.9523 29.262 27.112C30.3786 28.2718 31.1165 29.0407 31.3264 29.2409C32.4562 30.3178 34.2187 30.2386 35.2594 29.0687C36.3002 27.8989 36.2386 26.0603 35.1087 24.9833C34.8988 24.7832 34.102 24.0833 32.8936 23.0267C31.6851 21.9701 30.0794 20.5747 28.3125 19.0664C24.9232 16.1731 20.9627 12.8774 18.0356 10.6454L19.2562 9.26794C19.861 9.73638 20.7228 9.66735 21.2453 9.0801L22.27 7.93746C22.8273 7.31105 22.7846 6.32278 22.1796 5.74605L16.5888 0.408515C16.2864 0.12016 15.8863 -0.0150351 15.5036 0.00132414ZM15.5488 0.893512C15.7099 0.88665 15.87 0.940947 16.0009 1.06569L21.5917 6.40323C21.8534 6.65276 21.8629 7.05575 21.6218 7.32679L20.5971 8.46942C20.356 8.74045 19.9698 8.76589 19.708 8.51635L19.5725 8.3811C19.5544 8.35822 19.5343 8.33725 19.5122 8.31849L14.3135 3.37215C14.3086 3.36681 14.3036 3.36161 14.2984 3.35653L14.283 3.3465C14.2732 3.34089 14.2631 3.33568 14.2529 3.33088L14.1172 3.19C13.8555 2.94047 13.846 2.53748 14.0871 2.26645L15.0968 1.12381C15.2173 0.9883 15.3877 0.911563 15.5488 0.904671V0.893512ZM14.0419 4.32154L14.6899 4.94766L18.5928 8.67306L17.1161 10.3479C17.0998 10.3634 17.0846 10.3802 17.0707 10.398L15.8049 11.8381L14.2679 13.5599L9.71701 9.20841L14.0419 4.32469L14.0419 4.32154ZM8.14977 9.20526C8.31087 9.1984 8.47097 9.2527 8.60184 9.37745L8.73686 9.52269C8.75561 9.54012 8.77577 9.55582 8.79714 9.56962L13.9959 14.5316L14.0109 14.5472C14.0297 14.5647 14.0499 14.5804 14.0712 14.5942L14.1918 14.7194C14.4535 14.9689 14.463 15.3719 14.2219 15.6429L13.1972 16.7856C12.9561 17.0566 12.5849 17.0821 12.3232 16.8327L6.71748 11.4951C6.45572 11.2456 6.44621 10.8426 6.68734 10.5716L7.71204 9.42893C7.83261 9.29341 7.98794 9.21668 8.14905 9.20979L8.14977 9.20526ZM17.4324 11.3184C20.3384 13.5269 24.3275 16.8453 27.7546 19.771C29.5185 21.2767 31.1298 22.6613 32.3356 23.7156C33.5414 24.7698 34.3749 25.5019 34.5206 25.6409C35.3048 26.3885 35.3484 27.6465 34.6261 28.4584C33.9037 29.2704 32.698 29.3156 31.9136 28.568C31.7679 28.429 30.9784 27.6434 29.8642 26.4861C28.75 25.3289 27.2848 23.7835 25.6901 22.0876C22.5937 18.7948 19.0891 14.9484 16.7239 12.1324L17.4324 11.3184ZM2.51367 18.1902C2.41914 18.2022 2.33103 18.2461 2.26292 18.3153C2.19481 18.3844 2.15048 18.4749 2.13677 18.5728C2.12307 18.6707 2.14074 18.7706 2.18706 18.857C2.23339 18.9435 2.30579 19.0117 2.39312 19.0511C4.59302 20.1196 6.07028 21.2927 7.00427 24.8585C7.01187 24.9227 7.03267 24.9844 7.06526 25.0395C7.09785 25.0946 7.14148 25.1418 7.19317 25.1779C7.24487 25.2139 7.30342 25.238 7.36486 25.2484C7.4263 25.2589 7.48919 25.2555 7.54926 25.2385C7.60932 25.2215 7.66516 25.1912 7.71299 25.1498C7.76081 25.1084 7.7995 25.0568 7.82642 24.9985C7.85335 24.9402 7.86788 24.8765 7.86903 24.8119C7.87018 24.7472 7.85793 24.683 7.8331 24.6237C6.84784 20.8624 5.07075 19.3548 2.76987 18.2373C2.69112 18.1948 2.60169 18.1783 2.51367 18.1902ZM15.7446 20.9295C15.7344 20.9342 15.7243 20.9394 15.7145 20.945C13.8198 21.4851 12.2357 22.4141 11.0129 25.14C10.9816 25.1943 10.9616 25.2548 10.9542 25.3177C10.9469 25.3805 10.9524 25.4442 10.9704 25.5047C10.9883 25.5652 11.0183 25.6211 11.0585 25.6687C11.0986 25.7164 11.1479 25.7547 11.2032 25.7813C11.2586 25.8078 11.3187 25.8219 11.3796 25.8227C11.4405 25.8235 11.5009 25.811 11.5569 25.7859C11.6129 25.7609 11.6631 25.7239 11.7044 25.6773C11.7457 25.6307 11.7771 25.5757 11.7965 25.5157C12.9366 22.9741 14.1527 22.3156 15.9405 21.806C16.0462 21.773 16.1358 21.6995 16.1912 21.6003C16.2465 21.5012 16.2634 21.3838 16.2385 21.2722C16.2135 21.1606 16.1486 21.063 16.0569 20.9993C15.9651 20.9356 15.8535 20.9107 15.7446 20.9295ZM0.720808 22.2756C0.607835 22.2841 0.496748 22.2987 0.389287 22.3227C0.328673 22.3293 0.270062 22.349 0.217267 22.3806C0.164473 22.4122 0.118679 22.455 0.0828656 22.5062C0.0470523 22.5574 0.0220228 22.6159 0.00940546 22.6778C-0.00321185 22.7398 -0.00313387 22.8038 0.00963448 22.8656C0.0224028 22.9275 0.047575 22.9859 0.0835131 23.0371C0.119451 23.0882 0.16535 23.1309 0.218221 23.1623C0.271092 23.1938 0.329752 23.2134 0.390381 23.2197C0.45101 23.2261 0.512251 23.2192 0.570116 23.1993C1.04936 23.0932 2.05162 23.3042 2.95104 23.7159C3.85046 24.1275 4.66612 24.7636 4.86484 25.1246C4.88896 25.1825 4.92451 25.2345 4.96919 25.2772C5.01386 25.32 5.06667 25.3525 5.1242 25.3727C5.18173 25.3929 5.2427 25.4003 5.30317 25.3945C5.36364 25.3887 5.42226 25.3698 5.47524 25.3389C5.52823 25.3081 5.5744 25.2661 5.61076 25.2156C5.64713 25.1651 5.67289 25.1072 5.68637 25.0457C5.69986 24.9842 5.70076 24.9204 5.68904 24.8585C5.67731 24.7966 5.65321 24.738 5.6183 24.6863C5.20771 23.9404 4.29482 23.3583 3.29765 22.9019C2.42514 22.5025 1.51164 22.2143 0.720808 22.2756Z"
      fill="currentColor"
    />
  </svg>
);
