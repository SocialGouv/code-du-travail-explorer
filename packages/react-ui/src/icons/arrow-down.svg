<svg viewBox="0 0 18 18" fill="none">
    <path d="M1 6L9 12L17 6" stroke="currentColor" stroke-width="1.25" stroke-linecap="round" stroke-linejoin="round"/>
</svg>
