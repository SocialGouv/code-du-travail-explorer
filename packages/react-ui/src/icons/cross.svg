<svg fill="none" viewBox="0 0 8 8">
    <path d="M0.666748 7.33342L4.00008 4.00008M7.33342 0.666748L4.00008 4.00008M4.00008 4.00008L0.666748 0.666748M4.00008 4.00008L7.33342 7.33342" stroke="currentColor" stroke-linecap="round"/>
</svg>
