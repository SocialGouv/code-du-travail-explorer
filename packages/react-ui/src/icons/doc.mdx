---
name: Icons
menu: Component
---

import styled from "styled-components";
import { Playground, Props } from "docz";
import * as icons from ".";

# Icons

<Playground>
  {
    () => {
      return Object.entries(icons).map(([name, Icon]) => <div key={name}>
          <p>{name}</p>
          <Icon width="3rem" />
        </div>
      )
    }
  }
</Playground>
