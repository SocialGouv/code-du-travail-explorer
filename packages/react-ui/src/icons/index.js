// Bicolor icons
export { default as Agreement } from "./components/bicolor/Agreement";
export { default as Calculator } from "./components/bicolor/Calculator";
export { default as Constitution } from "./components/bicolor/Constitution";
export { default as Contract } from "./components/bicolor/Contract";
export { default as Custom } from "./components/bicolor/Custom";
export { default as Depart } from "./components/bicolor/Depart";
export { default as Document } from "./components/bicolor/Document";
export { default as Eurotext } from "./components/bicolor/Eurotext";
export { default as Focus } from "./components/bicolor/Focus";
export { default as Formation } from "./components/bicolor/Formation";
export { default as Health } from "./components/bicolor/Health";
export { default as Holidays } from "./components/bicolor/Holidays";
export { default as Indemnity } from "./components/bicolor/Indemnity";
export { default as International } from "./components/bicolor/International";
export { default as Laws } from "./components/bicolor/Laws";
export { default as Logo } from "./components/bicolor/Logo";
export { default as Nego } from "./components/bicolor/Nego";
export { default as Oups } from "./components/bicolor/Oups";
export { default as PartialActivity } from "./components/bicolor/PartialActivity";
export { default as Populars } from "./components/bicolor/Populars";
export { default as Precarity } from "./components/bicolor/Precarity";
export { default as PriorNotice } from "./components/bicolor/PriorNotice";
export { default as Resignation } from "./components/bicolor/Resignation";
export { default as Resources } from "./components/bicolor/Resources";
export { default as Rules } from "./components/bicolor/Rules";
export { default as Salary } from "./components/bicolor/Salary";
export { default as Security } from "./components/bicolor/Security";
export { default as Termination } from "./components/bicolor/Termination";
export { default as Time } from "./components/bicolor/Time";
export { default as Unemployment } from "./components/bicolor/Unemployment";
export { default as Uses } from "./components/bicolor/Uses";
export { default as Warning } from "./components/bicolor/Warning";
export { default as WorkersHero } from "./components/bicolor/WorkersHero";
export { default as WorkersTheme } from "./components/bicolor/WorkersTheme";

// Monochrome icons
export { default as ArrowDown } from "./components/monochrome/ArrowDown";
export { default as ArrowRight } from "./components/monochrome/ArrowRight";
export { default as Burger } from "./components/monochrome/Burger";
export { default as Calendar } from "./components/monochrome/Calendar";
export { default as External } from "./components/monochrome/Calendar";
export { default as Check } from "./components/monochrome/Check";
export { default as Close } from "./components/monochrome/Close";
export { default as Config } from "./components/monochrome/Config";
export { default as DirectionLeft } from "./components/monochrome/DirectionLeft";
export { default as DirectionRight } from "./components/monochrome/DirectionRight";
export { default as Download } from "./components/monochrome/Download";
export { default as Euro } from "./components/monochrome/Euro";
export { default as Help } from "./components/monochrome/Help";
export { default as Home } from "./components/monochrome/Home";
export { default as Link } from "./components/monochrome/Link";
export { default as Mail } from "./components/monochrome/Mail";
export { default as More } from "./components/monochrome/More";
export { default as Search } from "./components/monochrome/Search";
export { default as Shade } from "./components/monochrome/Shade";
export { default as Share } from "./components/monochrome/Share";
export { default as ShareFacebook } from "./components/monochrome/ShareFacebook";
export { default as ShareLinkedin } from "./components/monochrome/ShareLinkedin";
export { default as ShareTwitter } from "./components/monochrome/ShareTwitter";
export { default as ShareWhatsapp } from "./components/monochrome/ShareWhatsapp";
