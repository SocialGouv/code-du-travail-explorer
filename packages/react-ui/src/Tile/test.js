import React from "react";
import { render } from "@testing-library/react";

import { Time } from "../icons";
import { Tile } from ".";

describe("<Tile />", () => {
  it("renders a tile with text inside", () => {
    const { container } = render(
      <Tile title="Hello !">There is text inside</Tile>
    );
    expect(container).toMatchSnapshot();
    expect(container.getElementsByTagName("button").length).toBe(1);
  });
  it("renders a tile with a custom badge", () => {
    const { container } = render(
      <Tile title="Hello !">There is text inside</Tile>
    );
    expect(container).toMatchSnapshot();
    expect(container.getElementsByTagName("button").length).toBe(1);
  });
  it("renders a tile with an icon", () => {
    const { container } = render(
      <Tile title="Hello !" icon={Time}>
        There is text inside
      </Tile>
    );
    expect(container).toMatchSnapshot();
    expect(container.getElementsByTagName("button").length).toBe(1);
  });
  it("renders a tile with a stripe", () => {
    const { container } = render(
      <Tile title="Hello !" striped>
        There is text inside
      </Tile>
    );
    expect(container).toMatchSnapshot();
  });
  it("renders a wide tile with a subtitle", () => {
    const { container } = render(
      <Tile subtitle="This is a theme" title="Hello !">
        There is text inside
      </Tile>
    );
    expect(container).toMatchSnapshot();
    expect(container.getElementsByTagName("button").length).toBe(1);
  });
});
