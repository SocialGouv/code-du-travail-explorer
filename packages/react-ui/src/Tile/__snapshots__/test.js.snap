// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Tile /> renders a button tile 1`] = `
.c0 {
  position: relative;
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-flex: 1 1 100%;
  -ms-flex: 1 1 100%;
  flex: 1 1 100%;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
  -webkit-align-items: flex-start;
  -webkit-box-align: flex-start;
  -ms-flex-align: flex-start;
  align-items: flex-start;
  -webkit-box-pack: end;
  -webkit-justify-content: flex-end;
  -ms-flex-pack: end;
  justify-content: flex-end;
  box-sizing: border-box;
  margin: 0;
  padding: 1.25rem 1.5rem;
  color: #005994;
  font-weight: bold;
  font-size: 1.125rem;
  text-align: left;
  -webkit-text-decoration: none;
  text-decoration: none;
  background-color: #fff;
  border: 1px solid #c9d3df;
  border-radius: 0.25rem;
  cursor: pointer;
  -webkit-transition: 250ms all;
  transition: 250ms all;
  -webkit-appearance: none;
  -moz-appearance: none;
  appearance: none;
}

.c0:hover {
  border-color: #005994;
  box-shadow: 0 5px 10px 0 #ebeff3;
}

.c1 {
  display: block;
  max-width: 100%;
  padding-right: 0;
  overflow-y: auto;
}

<div>
  <button
    class="c0"
    href=""
  >
    <div
      class="c1"
    >
      Hello !
    </div>
    
  </button>
</div>
`;

exports[`<Tile /> renders dark and small 1`] = `
.c0 {
  position: relative;
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-flex: 1 1 100%;
  -ms-flex: 1 1 100%;
  flex: 1 1 100%;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
  -webkit-align-items: flex-start;
  -webkit-box-align: flex-start;
  -ms-flex-align: flex-start;
  align-items: flex-start;
  -webkit-box-pack: end;
  -webkit-justify-content: flex-end;
  -ms-flex-pack: end;
  justify-content: flex-end;
  box-sizing: border-box;
  margin: 0;
  padding: 1rem 1.25rem;
  color: #005994;
  font-weight: bold;
  text-align: left;
  -webkit-text-decoration: none;
  text-decoration: none;
  background-color: #f9f9fc;
  border: 1px solid #c9d3df;
  border-radius: 0.25rem;
  cursor: pointer;
  -webkit-transition: 250ms all;
  transition: 250ms all;
  -webkit-appearance: none;
  -moz-appearance: none;
  appearance: none;
}

.c0:hover {
  border-color: #005994;
  box-shadow: 0 5px 10px 0 #ebeff3;
}

.c1 {
  display: block;
  max-width: 100%;
  padding-right: 0;
  overflow-y: auto;
}

<div>
  <button
    class="c0"
    href=""
  >
    <div
      class="c1"
    >
      Hello !
    </div>
    
  </button>
</div>
`;

exports[`<Tile /> renders with a div fake button inside 1`] = `
.c3 {
  display: inline-block;
  padding: 0.625rem 1.5rem;
  font-weight: 600;
  font-size: 1rem;
  line-height: inherit;
  text-align: center;
  vertical-align: middle;
  border: 1px solid;
  border-radius: 1.5rem;
  cursor: pointer;
  -webkit-transition: background-color 250ms ease;
  transition: background-color 250ms ease;
  -webkit-appearance: none;
  -moz-appearance: none;
  appearance: none;
  color: #0c0c0e;
  background: #ebeff3;
  border-color: #ebeff3;
}

.c3:link,
.c3:visited {
  -webkit-text-decoration: none;
  text-decoration: none;
  color: #0c0c0e;
}

.c3:not([disabled]):hover,
.c3:not([disabled]):active,
.c3:not([disabled]):focus {
  color: #18181c;
  background: #fbfcfd;
  border-color: #dbe2e9;
}

.c3[disabled] {
  color: rgba(12,12,14,0.4);
  cursor: not-allowed;
}

.c0 {
  position: relative;
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-flex: 1 1 100%;
  -ms-flex: 1 1 100%;
  flex: 1 1 100%;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
  -webkit-align-items: flex-start;
  -webkit-box-align: flex-start;
  -ms-flex-align: flex-start;
  align-items: flex-start;
  -webkit-box-pack: end;
  -webkit-justify-content: flex-end;
  -ms-flex-pack: end;
  justify-content: flex-end;
  box-sizing: border-box;
  margin: 0;
  padding: 1.25rem 1.5rem;
  color: #005994;
  font-weight: bold;
  font-size: 1.125rem;
  text-align: left;
  -webkit-text-decoration: none;
  text-decoration: none;
  background-color: #fff;
  border: 1px solid #c9d3df;
  border-radius: 0.25rem;
  cursor: pointer;
  -webkit-transition: 250ms all;
  transition: 250ms all;
  -webkit-appearance: none;
  -moz-appearance: none;
  appearance: none;
}

.c0:hover {
  border-color: #005994;
  box-shadow: 0 5px 10px 0 #ebeff3;
}

.c1 {
  display: block;
  max-width: 100%;
  padding-right: 0;
  overflow-y: auto;
}

.c2 {
  margin-top: 1.25rem;
}

<div>
  <a
    class="c0"
    href="#"
  >
    <div
      class="c1"
    >
      Hello !
    </div>
    <div
      class="c2"
    >
      <div
        class="c3"
      >
        bouton
      </div>
    </div>
  </a>
</div>
`;

exports[`<Tile /> renders with an icon 1`] = `
.c0 {
  position: relative;
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-flex: 1 1 100%;
  -ms-flex: 1 1 100%;
  flex: 1 1 100%;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
  -webkit-align-items: flex-start;
  -webkit-box-align: flex-start;
  -ms-flex-align: flex-start;
  align-items: flex-start;
  -webkit-box-pack: end;
  -webkit-justify-content: flex-end;
  -ms-flex-pack: end;
  justify-content: flex-end;
  box-sizing: border-box;
  margin: 0;
  padding: 1.25rem 1.5rem;
  color: #005994;
  font-weight: bold;
  font-size: 1.125rem;
  text-align: left;
  -webkit-text-decoration: none;
  text-decoration: none;
  background-color: #fff;
  border: 1px solid #c9d3df;
  border-radius: 0.25rem;
  cursor: pointer;
  -webkit-transition: 250ms all;
  transition: 250ms all;
  -webkit-appearance: none;
  -moz-appearance: none;
  appearance: none;
}

.c0:hover {
  border-color: #005994;
  box-shadow: 0 5px 10px 0 #ebeff3;
}

.c2 {
  display: block;
  max-width: 100%;
  padding-right: 1rem;
  overflow-y: auto;
}

.c1 {
  position: absolute;
  top: 0.5rem;
  right: 0.5rem;
}

<div>
  <button
    class="c0"
    href=""
  >
    <div
      class="c1"
    >
      <svg
        fill="none"
        height="24"
        stroke="currentColor"
        stroke-linecap="round"
        stroke-linejoin="round"
        stroke-width="2"
        viewBox="0 0 24 24"
        width="24"
        xmlns="http://www.w3.org/2000/svg"
      >
        <line
          x1="8"
          x2="21"
          y1="6"
          y2="6"
        />
        <line
          x1="8"
          x2="21"
          y1="12"
          y2="12"
        />
        <line
          x1="8"
          x2="21"
          y1="18"
          y2="18"
        />
        <line
          x1="3"
          x2="3"
          y1="6"
          y2="6"
        />
        <line
          x1="3"
          x2="3"
          y1="12"
          y2="12"
        />
        <line
          x1="3"
          x2="3"
          y1="18"
          y2="18"
        />
      </svg>
    </div>
    <div
      class="c2"
    >
      Hello !
    </div>
    
  </button>
</div>
`;
